tests/unit/backend/wmg/fixtures/test_snapshot.py

```diff
@@ -218,6 +218,50 @@ def load_realistic_test_snapshot(snapshot_name: str) -> WmgSnapshot:
             )
 
 
+def load_realistic_test_snapshot_obj(snapshot_name: str, cube_dir: str) -> WmgSnapshot:
```

Comment:
this function is needed to be able to patch "load_snapshot" since `load_snapshot` returns WmgSnapshot whereas `load_realistic_test_snapshot` returns a context manager.